### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Reflow: Unable to see all controls and content in reflow

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2698]
[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev2; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Non-Text Contrast; #WCAG1.4.10; #Win10;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0

OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### NOTE:
Issue is observed through out the application.

### Repro Steps:

1. Set the display resolution to 1280*1024 and advanced scaling to 400% and sign out and launch the application.
2. Open Visual studio code
3. Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Verify user is able to see all controls or not

### Actual Result:
Unable to see all controls and content in reflow 

### Expected Result:
User should be able to see all controls and content on screen using only one scroll bar

### User Impact:
Low vision users will find difficulty in viewing the content and to perform operations if all controls are not visible in reflow

### Attachment for Reference:
![2698_After Reflow](https://user-images.githubusercontent.com/38523250/108504657-3f947e80-72dc-11eb-89eb-03f0dfde2e1b.png)

**Comment by ejizba** [comment]
Our extension does not control zooming of the UI. Tracked in VS Code by https://github.com/microsoft/vscode/issues/115878
